help/en/html/tools/logixng/reference/chapter5.shtml

```diff
@@ -213,13 +213,18 @@
 
       <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
 
-      <h4>Clock</h4>
+      <h4>Clock<span class="since">updated 4.99.7</span></h4>
+
+      <div style="margin-left: 2em;">
+        <a href="images/chapter5/clock_action.png"><img src="images/chapter5/clock_action.png"
+        alt="Chapter 5 clock action"></a>
+      </div>
 
       <ul>
         <li><strong>Set Fast Clock:</strong> Sets the time on the fast clock to the entered time.
-        The time is entered as hh:mm where hh is hours and mm is minutes on a 24-hour clock. This
-        action sets the fast clock to the specified time regardless of whether the fast clock is
-        running or stopped.</li>
+        The time can entered as hh:mm where hh is hours and mm is minutes on a 24-hour clock or as
```

Comment:
I'm not a native English speeker, but I'm wondering if the sentense should be:
The time can `be` entered as hh:mm

Reply:
Thanks.